app/client/keybase/keybase.go

```diff
@@ -40,3 +43,44 @@ type Keybase interface {
 	// Removals
 	Delete(address, passphrase string) error
 }
+
+type KeybaseType int
+
+const (
+	KeybaseTypeFile KeybaseType = iota
+	KeybaseTypeVault
+)
+
+type KeybaseOptions struct {
```

Comment:
Per my other comments, I'm beginning to think that this should be part of `config.json`, which configures our modules, rpc, etc.. since it's part of a node's config.

If you look at `config.proto` it should be a good start.

Also, even though I think this is the right approach, I'm not married to it and happy to hear counterarguments.

Reply:
💯 I was a bit confused finding that the CLI and node keybase logic were disconnected. Though, I think that should be tackled in a separate PR especially since there aren't any unit tests on the CLI commands.